### ISSUE TITLE
Issue with getting some content

### ISSUE THREAD

**Issue by Darkedel** [whosneksio/wuwa.api#3]
the content for weapons: 
Gauntlets#21D
Originite: Type V
Rectifier#25
Training Rectifier
Pistols#26
Originite: Type III
Originite: Type II
Sword#18
Broadblade#41
Scale: Slasher

are giving the error: Weapon not found

**Comment by whosneksio** [comment]
yeah i know, ill try to fix them asap but the weapon are still in progress

**Comment by Darkedel** [comment]
Ok thank you for letting me know
